hub/tests/test_sources.py

```diff
@@ -0,0 +1,120 @@
+from django.test import TestCase
+from django.conf import settings
+from datetime import datetime
+
+from hub.models import AirtableSource, ExternalDataSourceUpdateConfig
+
+
+class TestDataSource(TestCase):
```

Comment:
It would be cool to have a bit in the `README.md` that explains our reasoning for running these tests end to end, which I think is sound.